spec/controllers/admins_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+describe AdminsController do
+  describe '#assign' do
+    def go!
+      post 'assign'
+    end
+
+    context 'logged out' do
+      it 'redirects' do
+        go!
+        expect(response).to redirect_to('/')
+      end
+    end
+
+    context 'logged in' do
+      let!(:admin) { FactoryGirl.create(:admin, :activated) }
+      let!(:grant) { FactoryGirl.create(:grant) }
+      let!(:grant_2) { FactoryGirl.create(:grant) }
+
+      before do
+        FactoryGirl.create_list(:grant_submission, 16, funding_decision: false, grant: grant)
```

Comment:
"This" refers to lines 20 to 35 which create the relevant models for the test following it.